.travis.yml

```diff
@@ -10,25 +10,15 @@ cache:
 
 language: scala
 
-scala:
-   - 2.11.8
-
 jdk:
    - oraclejdk8
 
-env:
-  global:
-    - SCALA_VERSION=2.11.8
-
 matrix:
   include:
-    - env: PYSPARK_PYTHON=python2 SPARK_VERSION=2.3.4 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
-    - env: PYSPARK_PYTHON=python2 SPARK_VERSION=2.4.4 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
-    - env: PYSPARK_PYTHON=python3 SPARK_VERSION=2.3.4 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
-    - env: PYSPARK_PYTHON=python3 SPARK_VERSION=2.4.4 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
-
-before_install:
- - ./bin/download_travis_dependencies.sh
+    - env: PYSPARK_PYTHON=python2 SCALA_VERSION=2.11.8 SPARK_VERSION=2.4.4 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
+    - env: PYSPARK_PYTHON=python2 SCALA_VERSION=2.12.10 SPARK_VERSION=3.0.0-SNAPSHOT SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
```

Comment:
I'd recommend we drop this since Spark 3.0 is expected to deprecate python 2 support.